wordpress-mobile/gutenberg-mobile
# Release 1.16.1

Prepares release 1.16.1

WP-iOS PR: https://github.com/wordpress-mobile/WordPress-iOS/pull/12897
WP-Android: https://github.com/wordpress-mobile/WordPress-Android/pull/10750
WP-GB PR: Not needed because there are no gutenberg side changes.

To test:
 - Smoke test all functionality for a new release
 - Pay special attention to the multiline text blocks and see if the update their size when empty line are inserted or deleted.

Update release notes:
